src/txmempool.h

```diff
@@ -585,6 +586,25 @@ class CTxMemPool
      */
     std::set<uint256> m_unbroadcast_txids GUARDED_BY(cs);
 
+
+    /**
+     * Helper function to calculate all in-mempool ancestors of staged_ancestors and apply
```

Comment:
```suggestion
     * Helper function to calculate all in-mempool ancestors of staged_ancestors (including staged_ancestors themselves) and apply
```

or similar to document that setAncestors will include the transactions in staged_ancestors.

Reply:
Done